standard/types.md

```diff
@@ -600,9 +601,20 @@ A type parameter is an identifier designating a value type or reference type tha
 ```ANTLR
 type_parameter
     : identifier
+
+nullable_type_parameter
+    : non_nullable_non_value_type_parameter '?'
+    ;
+
+non_nullable_non_value_type_parameter
+    : type_parameter
     ;
 ```
 
+The *non_nullable_non_value_type_parameter* in *nullable_type_parameter* shall be a type parameter that isn’t constrained to be a value type.
+
+In *nullable_type_parameter*, the annotation `?` indicates the intent that nullable type corresponding to the type arguments of this type are nullable. The absence of the annotation `?` indicates the intent that type arguments of this type are non-nullable.
```

Comment:
This seems to imply that the *type_parameter_list* of a generic declaration can contain *nullable_type_parameter* but that is not (currently?) the case. *nullable_type_parameter* is a *type* and used in such things as the declaration of variables.

Is not the intent described here conveyed by the constraints §15.2.5?

Suggestion: delete this

```suggestion
```

Reply:
done (in next commit)